android/KMAPro/kMAPro/src/main/java/com/keyman/android/SystemKeyboard.java

```diff
@@ -140,6 +140,17 @@ public void onUpdateSelection(int oldSelStart, int oldSelEnd, int newSelStart, i
     KMManager.updateSelectionRange(KMManager.KeyboardType.KEYBOARD_TYPE_SYSTEM);
   }
 
+  private void sendCurrentFontName() {
+    Keyboard keyboardInfo = KMManager.getCurrentKeyboardInfo(this);
+    if (keyboardInfo != null) {
+      String fontName = keyboardInfo.getFont();
+
+      Intent intent = new Intent("com.keyman.android.keyboard_changed");
```

Comment:
Do we need to document this Intent in the API documentation, @darcywong00?

Reply:
I think it belongs more closely with Engine docs than end user docs. It's about integration, just at another level. Let's go with putting it into the Engine docs, under a new section. If we have to reorg Engine docs, that's also okay.